provisioning/miniprov/hare_mp/main.py

```diff
@@ -224,13 +285,13 @@ def checkRpm(rpm_name):
                                 stdin=subprocess.PIPE,
                                 stdout=subprocess.PIPE,
                                 stderr=subprocess.PIPE,
-                                env={},
+                                env=None,
                                 encoding='utf8')
     rpm_search = subprocess.Popen(["grep", "-q", rpm_name],
                                   stdin=rpm_list.stdout,
                                   stdout=subprocess.PIPE,
                                   stderr=subprocess.PIPE,
-                                  env={},
+                                  env=None,
```

Comment:
What is the meaningful difference between `env=None` and `env={}`?

If `env=None` makes subprocess to inherit the environment of the current process, then it is worth removing `env=` kwarg at all.

Reply:
Done, default value of env in None anyway. In that case it will inherit parent